### PR TITLE
Add payment instrument get/patch endpoint

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

File: openapi/core.yaml [hunk 1]
```diff
@@ -379,6 +384,8 @@ paths:
     $ref: './paths/paypal-accounts@{id}@activation.yaml'
   '/paypal-accounts/{id}/deactivation':
     $ref: './paths/paypal-accounts@{id}@deactivation.yaml'
+  '/payment-instruments/{id}':
+    $ref: './paths/payment-instruments@{id}.yaml'
```
Comment (justnero) [inline openapi/core.yaml:388]: We need to have it in combined API definitions as well

**Comment by ashkarpetin** [comment]
I can't find `Payment instruments` in preview: https://preview.redoc.ly/odd-puma-69/patch-payment-instrument/

File: openapi/combined.yaml [hunk 2]
```diff
@@ -92,6 +92,10 @@ tags:
     description: >
       Invoices leave a record for both you and your customer of the products
       sold.
+  - name: Payment Instruments
+    description: >
+      Payment instrument use for transaction and can pass
+      it onward securely to any gateway account.
```
Comment (ashkarpetin) [inline openapi/combined.yaml:98]: The wording seem incorrect here..

File: openapi/components/schemas/PaymentInstruments/PaymentInstrument.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+oneOf:
+- $ref: ../PaymentCards/PaymentCard.yaml
+- $ref: ../Common/CommonBankAccount.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/PaymentInstruments/PaymentInstrument.yaml:3]:
```suggestion
- $ref: ../BankAccounts/BankAccount.yaml
```

File: openapi/components/schemas/PaymentInstruments/PaymentInstrument.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+- $ref: ../Common/CommonPayPalAccount.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/PaymentInstruments/PaymentInstrument.yaml:4]:
```suggestion
- $ref: ../PayPalAccount.yaml
```

**Review by ashkarpetin — APPROVED** [review]

**Review by iamdevice — APPROVED** [review]

**Pull request merged**